### PR TITLE
Fix duplicated identifier BlockModuleOptions

### PR DESCRIPTION
**Pull request by dkarnutsch**

### CODE REVIEW

File: packages/api/cms-api/src/blocks/blocks.module.ts [hunk 1]
```diff
@@ -8,13 +8,13 @@ import { BlocksTransformerService } from "./blocks-transformer.service";
 import { CommandsService } from "./commands.service";
 import { DiscoverService } from "./discover.service";
 
-export interface BlocksModuleOptions {
+export interface BlocksModuleFactoryOptions {
```
Comment (johnnyomair) [inline packages/api/cms-api/src/blocks/blocks.module.ts:11]: I'm not sure about the naming. Maybe it should be `BlocksModuleOptions` for the actual options, and `BlocksModuleSyncOptions` for `forRoot`, and `BlocksModuleAsyncOptions` for `forRootAsync`. At least that's how MikroORM does it: https://github.com/mikro-orm/nestjs/blob/master/src/mikro-orm.module.ts#L20

This isn't about this PR in particular, rather a general concern.

Comment (nsams) [reply packages/api/cms-api/src/blocks/blocks.module.ts:11]: :+1:

Comment (nsams) [reply packages/api/cms-api/src/blocks/blocks.module.ts:11]: for most other modules I removed one of the two interfaces, for this one I made a mistake...

**Comment by nsams** [comment]
i intergrated this into #989, to have one branch I can depend on

File: packages/api/cms-api/src/blocks/blocks.module.ts (hunk 1 above)
Comment (johnnyomair) [reply packages/api/cms-api/src/blocks/blocks.module.ts:11]: I think it's interesting that the linter didn't complain about it either

**Pull request closed without merging**